src/data/map.rs.in

```diff
@@ -0,0 +1,32 @@
+use id::{Id, HasForgeableId, HasId};
+use uuid::Uuid;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Map {
+    pub id: Id<Map>,
```

Comment:
This one is a bit trickier because it is actually not defined in the RSP (i.e. this structure is used nowhere in the management API).

I intend to store everything that the server needs to know about a map in this structure (so for example which parts block the movement), with the layout found in Gjanajo (not defined yet)

What is returned as part of the REST API is the `GetMaps` structure defined in `src/data/management.rs.in`

Still, it doesn't hurt to be consistent and use a `uuid` field :)